laravel/resources/js/ui/userSearch.js

```diff
@@ -0,0 +1,75 @@
+var $ = require('wetfish-basic');
+var ajaxOptions =
+{
+    method: 'post',
+    credentials: 'include',
+    headers:
+    {
+        'Content-Type': 'application/json'
+    }
+};
+
+$(document).ready(function()
+{
+    $('.user-search input').on('keydown', function(event)
+    {
+        console.log('enter key pressed');
+
+        // If enter was pressed
+        if(event.which == 13)
+        {
+            event.preventDefault();
+            
+            $('.user-search').trigger('submit');
+        }
+    });
+
+    $('.user-search button').on('click', function(event)
+    {
+        event.preventDefault();
+        console.log('search button clicked');
```

Comment:
Using `console.log` this way should only be done for testing and not committed into git